Dockerfile-test

```diff
@@ -0,0 +1,5 @@
+FROM node:16-alpine3.11
```

Comment:
Do we really need another dockerFile specific to test?

Reply:
i tried to use dockerfile but it showed some error.Thats why created new Dockerfile for test